src/containers/Tenant/Tenant.tsx

```diff
@@ -77,10 +77,13 @@ function Tenant(props: TenantProps) {
     const {name} = queryParams;
     const tenantName = name as string;
 
+    useEffect(() => {
+        dispatch(getSchema({path: tenantName}));
+    }, [dispatch, tenantName]);
+
     useEffect(() => {
         const schemaPath = currentSchemaPath || tenantName;
         dispatch(resetLoadingState());
-        dispatch(getSchema({path: tenantName}));
         dispatch(getSchema({path: schemaPath}));
```

Comment:
If `currentSchemaPath` is undefined, `getSchema` in this useEffect will be fired with `tenantName` and the same will be done in previous useEffect.